test/evergreen.yml

```diff
@@ -1638,6 +1662,105 @@ tasks:
+          script: |
+            set -o errexit
+            set -o verbose
+            ${test_env_vars|} $(pwd)/../test/csuite/random_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/test_truncated_log 2>&1
+
+  - name: check-test-long-2
+    tags: ["check-long-test"]
+    commands:
+      - func: "get project"
+      - func: "configure wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-g -Werror"
+          posix_configure_flags: --enable-silent-rules --enable-diagnostic --disable-static
+      - func: "make wiredtiger"
+        vars:
+          make_command: make all
+      - func: "make check all"
+        vars:
+          test_env_vars: TESTUTIL_ENABLE_LONG_TESTS=1
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            WT3363_CHECKPOINT_OP_RACES=1 $(pwd)/test/csuite/./test_wt3363_checkpoint_op_races
+      - func: "many dbs test"
+      - func: "many dbs test"
+        vars:
+          many_db_args: -I
+      - func: "many dbs test"
+        vars:
+          many_db_args: -D 40
+      - func: "many dbs test"
+        vars:
+          many_db_args: -I -D 40
+      - func: "thread check long test"
+        vars: 
+          thread_check_args: -t f
+      - func: "thread check long test"
+        vars: 
+          thread_check_args: -S -F -n 100000 -t f
+      - func: "thread check long test"
+        vars: 
+          thread_check_args: -t r
+      - func: "thread check long test"
+        vars: 
+          thread_check_args: -S -F -n 100000 -t r
+      - func: "thread check long test"
+        vars: 
+          thread_check_args: -t v
+      - func: "thread check long test"
+        vars:
+          thread_check_args: -S -F -n 100000 -t v
+      - func: "random abort test"
+      - func: "random abort test"
+        vars:
+          random_abort_args: -t 10
+      - func: "random abort test"
+        vars:
+          random_abort_args: -t 40
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/test/csuite/"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            ./test_truncated_log
+      - func: "test format"
+        vars:
+          extra_args: file_type=fix
+      - func: "test format"
+        vars:
+          extra_args: file_type=row
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix/bench/wtperf"
+          script: |
+            rundir=../../../bench/wtperf/runners
+            $rundir/wtperf_run.sh $rundir/mongodb-oplog.wtperf
+            num_truncates=`grep "^Truncate count:" ./wtperf.out | cut -d ' ' -f 3`
```

Comment:
Those comments and echo lines from the Jenkins job are useful and should be retained.